### ISSUE TITLE
Support for ING-DiBa [DE]

### ISSUE THREAD

**Issue by torbengb** [torbengb/bank2ynab#152]
This is specified as part of #148 and I have already captured it as `[DE ING-DiBa]`.

But the problem is that the actual CSV file appears to have **seven header lines**, not just one!  
[sample CSV file](https://github.com/DarkSmile92/YNABConverterPy/blob/master/DE_ING-DiBa_Sample_DarkSmile92.csv)

* @nocalla @toyg Can we handle that without writing a plugin?

**Comment by torbengb** [comment]
This hurdle can be solved with issue #153 .

**Comment by nocalla** [comment]
Proposed changed config:
```
[DE ING-DiBa]
# Issue #152 (via #148)
Use Regex For Filename = True
Source Filename Pattern = Umsatzanzeige_[0-9]{10}_[0-9]{8}
Source CSV Delimiter = ;
Header Rows = 7
#Buchung;Valuta;Auftraggeber/Empfänger;Buchungstext;Verwendungszweck;Betrag;Währung;Saldo;Währung
Input Columns = skip,skip,Payee,Memo,skip,Inflow,skip,skip,skip
```
